### PR TITLE
flutter: add create example

### PR DESCRIPTION
**Pull request by hellojukay**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

Can you move the help command to last?

**Review by kbdharun — APPROVED** [review]

File: pages/common/flutter.md [hunk 1]
```diff
@@ -3,9 +3,9 @@
 > Google's free, open source, and cross-platform mobile app SDK.
 > More information: <https://github.com/flutter/flutter/wiki/The-flutter-tool>.
 
-- Display help about a specific command:
+- Create a flutter project in current directory:
```
Comment (navarroaxel) [inline pages/common/flutter.md:6]: `In the current directory` or `in a directory of the same name`?

Comment (hellojukay) [reply pages/common/flutter.md]: The command is  `flutter create {{directory_path}}`, so i think `Create a flutter project in specific directory` is better.

File: pages/common/flutter.md [hunk 2]
```diff
@@ -3,9 +3,9 @@
 > Google's free, open source, and cross-platform mobile app SDK.
 > More information: <https://github.com/flutter/flutter/wiki/The-flutter-tool>.
 
-- Display help about a specific command:
+- Create a flutter project in specific directory:
 
-`flutter help {{command}}`
+`flutter create {{directory_path}}`
```
Comment (navarroaxel) [inline pages/common/flutter.md:8]:
```suggestion
`flutter create {{path/to/directory}}`
```

File: pages/common/flutter.md [hunk 3]
```diff
@@ -3,9 +3,9 @@
 > Google's free, open source, and cross-platform mobile app SDK.
 > More information: <https://github.com/flutter/flutter/wiki/The-flutter-tool>.
 
-- Display help about a specific command:
+- Create a flutter project in specific directory:
```
[the remark this answers is not in the document]
Comment (hellojukay) [reply pages/common/flutter.md]: OK, I respect your opinion.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**